src/styles/card.css

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  font-family: Roboto, sans-serif;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #e5e5e5;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  justify-content: center;
+  align-items: center;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
```

Comment:
Do you need flex properties without display: flex? )

Reply:
corrected